packages/webpack-configs/src/defineConfig.ts

```diff
@@ -292,7 +290,7 @@ export function defineDevRemoteModuleConfig(swcConfig: SwcConfig, applicationNam
     const {
         entry = path.resolve("./src/register.tsx"),
         publicPath = "auto",
-        cache = false,
+        cache = true,
```

Comment:
It's true by default so you don't need to specifically specify a value.

Reply:
Fixed